sdcm/cluster.py

```diff
@@ -509,6 +510,10 @@ def running_nemesis(self, nemesis):
         with self.lock:
             self._running_nemesis = nemesis
 
+    @property
+    def wait_for_stop_event(self):
+        return self._wait_for_stop_event
```

Comment:
I cant see the reason for @property there. Can we use it the same nanner as  self.termination_event?

Reply:
fixed